### PR TITLE
Talos - Bump react, react-test-renderer, react-dom...

### PR DESCRIPTION
**Pull request by bbc-news-frameworks**
👋 The following packages have been published:
react
react-dom
react-test-renderer

So we need to bump them in the following packages:
psammead
@bbc/psammead-amp-geo
@bbc/psammead-image
@bbc/psammead-image-placeholder
@bbc/psammead-navigation
@bbc/psammead-media-indicator
@bbc/psammead-consent-banner
@bbc/psammead-story-promo-list
@bbc/psammead-assets
@bbc/psammead-sitewide-links
@bbc/psammead-media-player
@bbc/psammead-section-label
@bbc/psammead-story-promo
@bbc/psammead-brand
@bbc/psammead-timestamp-container

### CODE REVIEW

**Review by dr3 — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
Resolve conflicts

**Review by ghost — APPROVED** [review]

File: packages/components/psammead-media-player/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-media-player",
-  "version": "1.0.1-alpha.1",
+  "version": "1.0.1",
```
Comment (ghost) [inline packages/components/psammead-media-player/package.json]: Bumping an alpha version to stable.
```suggestion
  "version": "1.0.1-alpha.2",
```
I have a PR to fix the alpha versioning https://github.com/bbc/psammead/pull/1949

**Review by ghost — DISMISSED** [review]
Just noticed the `alpha-version` bump.

**Review by jroebu14 — APPROVED** [review]

**Comment by staylos92** [comment]
LGTM 👍

**Pull request merged**